pkg/operator/configobservation/apiserver/observe_apiserver.go

```diff
@@ -62,17 +62,6 @@ var ObserveUserClientCABundle configobserver.ObserveConfigFunc = (&apiServerObse
 	resourceType:  corev1.ConfigMap{},
 }).observe
 
-// ObserveDefaultUserServingCertificate returns an ObserveConfigFunc that observes user managed TLS cert info for
-// serving secure traffic.
-// We are disabling this because it doesn't work today and customers aren't going to be able to get the kube service network options right.
-// Customers may only use SNI.  I'm leaving this code in case we ever come up with a way to make an SNI-like thing based on IPs.
-var ObserveDefaultUserServingCertificate configobserver.ObserveConfigFunc = (&apiServerObserver{
```

Comment:
why is this commit good? removal without explanation in the commit msg is not good.

Reply:
this is bump of openshift/api not something this PR introduced

Reply:
but it changes pkg/operator/configobservation/apiserver/observe_apiserver.go, not vendor/openshift/api.

Reply:
this was a backport of breaking api change which just disabled the code, but left it there. this PR won the first bump and dealing with incompatibilities